lib/cylc/taskdef.py

```diff
@@ -36,15 +38,18 @@ def __str__(self):
 class TaskDef(object):
     """Task definition."""
 
+    # Store the elapsed times for a maximum of 100 cycles
```

Comment:
I'd also prefer if the variable here was created and set *after* the `__slots__` call